config/routes.rb

```diff
@@ -5,4 +5,5 @@
   post 'sessions', to: 'sessions#create'
   delete 'sessions', to: 'sessions#destroy' 
   root 'users#new'
+  get 'users(/:id)', to: 'users#show' 
```

Comment:
Question for my understanding: How come this is needed?

Reply:
![image](https://user-images.githubusercontent.com/60652902/81978650-bef57300-9623-11ea-8550-be5f0199d298.png)